src/status_im/subs.cljs

```diff
@@ -57,6 +57,10 @@
 (reg-root-key-sub :bottom-sheet/view :bottom-sheet/view)
 (reg-root-key-sub :bottom-sheet/options :bottom-sheet/options)
 
+(reg-root-key-sub :bottom-sheet-redesign/show? :bottom-sheet-redesign/show?)
```

Comment:
could you elaborate on why bottom sheet redesing is needed ?

Reply:
i will just use  new-ui-enabled